hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -451,3 +451,10 @@ message ContainerBalancerConfigurationProto {
     required bool shouldRun = 18;
     optional int32 nextIterationIndex = 19;
 }
+
+message DeletedBlocksTransactionInfo {
+    required int64 txID = 1;
```

Comment:
In general, we should let the proto layer mark args as optional and deal with it on the server side? I would mark most of these args as optional. cc @errose28

Reply:
@kerneltime I have a little confusion here, this ```DeletedBlocksTransactionInfo``` is returned by the server, and the client now would never construct it as request args.

Reply:
Got it, I have updated it. @kerneltime